README.rst

```diff
@@ -59,3 +62,34 @@ This will create a temporary directory called tdir and cd into it then run both
 and put their results inside of their own directory named after themselves.
 
 From there you can explore them on your own
+
+Running via Docker
+^^^^^^^^^^^^^^^^^^
+
```

Comment:
You need to add a comment that:
1. They need to have docker installed
2. If they don't have docker installed post the link they need to go to in order install it.